#!/usr/bin/env python

PACKAGE = "pointcloud_ros_wrapper"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("cam_angle",  double_t, 0, "camera angle(unit: Degree) parameter", 30, 0, 60)
gen.add("frontend",   double_t, 0, "ROI frontend parameter", 10, 0, 200)
gen.add("backend",    double_t, 0, "ROI backend parameter", 100, 0, 200)

exit(gen.generate(PACKAGE, "pointcloud_segment", "segment"))
